laboratory/src/pages/index.tsx

```diff
@@ -43,6 +43,12 @@ const cards = [
     description: 'Simple ethereum example that uses WalletConnect Auth SDK',
     link: '/v2Auth',
     color: 'success'
+  },
+  {
```

Comment:
Let's remove this to keep examples to full modal ones with differen't configs, but new component and it's props should be documented in

https://docs.walletconnect.com/2.0/web3modal/react/components
and
https://docs.walletconnect.com/2.0/web3modal/html-js/components

Reply:
Will do